### ISSUE TITLE
VC unresponsive after longevity 6.0 failure

### ISSUE THREAD

**Issue by mhagen-vmware** [vmware/vic#6670]
```
govc about
govc: expected element type <Envelope> but have <TITLE>
```

![screenshot from 2017-11-01 14-36-19](https://user-images.githubusercontent.com/19393273/32293749-219e116a-bf12-11e7-9b39-a94d98eb1fde.png)

**Comment by mhagen-vmware** [comment]
Gathering the vc-support bundle didn't go great:
```
19:38:54: Creating /storage/log/vc-bart-2017-11-01--19.38.tgz
19:38:55: Gathering output from /usr/bin/waiter-cfg dbdump /etc/vmware-rbd/autod19:38:55: Adding /storage/log/vmware/rbd/rbd-syslog.log                         19:38:55: Gathering output from /usr/sbin/iptables -L -n    Exception in thread Thread-2:
Traceback (most recent call last):
  File "/usr/lib64/python2.6/threading.py", line 532, in __bootstrap_inner
    self.run()
  File "/usr/lib/vmware/site-packages/vmsupport/vmtar.py", line 187, in run
    self.runInt()
  File "/usr/lib/vmware/site-packages/vmsupport/vmtar.py", line 157, in runInt
    self._tarObj.addFromBuf(self._tarPath + suffix, dataChunk)
  File "/usr/lib/vmware/site-packages/vmsupport/vmtar.py", line 373, in addFromBuf
    self._archiver.addFromBuf(self.generateUniqDestinationPath(name), data, mode)
  File "/usr/lib/vmware/site-packages/vmsupport/archiver.py", line 103, in addFromBuf
    self._tar.addfile(tarinfo=ti, fileobj=dataStream)
  File "/usr/lib64/python2.6/tarfile.py", line 2010, in addfile
    copyfileobj(fileobj, self.fileobj, tarinfo.size)
  File "/usr/lib64/python2.6/tarfile.py", line 288, in copyfileobj
    dst.write(buf)
  File "/usr/lib64/python2.6/gzip.py", line 197, in write
    self.fileobj.write( self.compress.compress(data) )
IOError: [Errno 28] No space left on device

19:38:55: Adding /storage/log/vmware/cis-license/vmware-cis-license-gc.log.1.cur19:38:55: Adding /storage/log/vmware/cis-license/vmware-cis-license-gc.log.2    19:38:56: Adding /storage/log/vmware/cis-license/vmware-cis-license-gc.log.4.cur19:38:56: Adding /storage/log/vmware/cis-license/vmware-cis-license-gc.log.5.cur19:38:56: Adding /storage/log/vmware/cis-license/wrapper.log                    19:38:56: Gathering output from /usr/lib/cloudvm-perfmon/cloudvm-perfmon.sh -z    Exception in thread Thread-3:
Traceback (most recent call last):
  File "/usr/lib64/python2.6/threading.py", line 532, in __bootstrap_inner
    self.run()
  File "/usr/lib/vmware/site-packages/vmsupport/vmtar.py", line 187, in run
    self.runInt()
  File "/usr/lib/vmware/site-packages/vmsupport/vmtar.py", line 157, in runInt
    self._tarObj.addFromBuf(self._tarPath + suffix, dataChunk)
  File "/usr/lib/vmware/site-packages/vmsupport/vmtar.py", line 373, in addFromBuf
    self._archiver.addFromBuf(self.generateUniqDestinationPath(name), data, mode)
  File "/usr/lib/vmware/site-packages/vmsupport/archiver.py", line 103, in addFromBuf
    self._tar.addfile(tarinfo=ti, fileobj=dataStream)
  File "/usr/lib64/python2.6/tarfile.py", line 2010, in addfile
    copyfileobj(fileobj, self.fileobj, tarinfo.size)
  File "/usr/lib64/python2.6/tarfile.py", line 282, in copyfileobj
    dst.write(buf)
  File "/usr/lib64/python2.6/gzip.py", line 197, in write
    self.fileobj.write( self.compress.compress(data) )
IOError: [Errno 28] No space left on device

19:39:08: Gathering output from /usr/lib/vmware-cm/bin/cmlookup -Dcm.url=http://localhost:18090/cm/sdk -Dprop=/usr/lib/vmware-cm/conf/cm.properties19:39:14: Adding /etc/vmware-eam/logging.properties                                                                                                19:39:14: Gathering output from /usr/lib/vmware-invsvc/scripts/exec-java.sh com.vmware.cis.lotus.LdapExport /var/log/vmware/invsvc/authz-ldapdump.l19:39:16: Adding /storage/log/vmware/invsvc/isp-SPS-d0bbe7f.log                                                                                    19:41:20: Gathering output from /bin/netstat -anp        Exception in thread Thread-12:                         
Traceback (most recent call last):
  File "/usr/lib64/python2.6/threading.py", line 532, in __bootstrap_inner
    self.run()
  File "/usr/lib/vmware/site-packages/vmsupport/vmtar.py", line 187, in run
    self.runInt()
  File "/usr/lib/vmware/site-packages/vmsupport/vmtar.py", line 157, in runInt
    self._tarObj.addFromBuf(self._tarPath + suffix, dataChunk)
  File "/usr/lib/vmware/site-packages/vmsupport/vmtar.py", line 373, in addFromBuf
    self._archiver.addFromBuf(self.generateUniqDestinationPath(name), data, mode)
  File "/usr/lib/vmware/site-packages/vmsupport/archiver.py", line 103, in addFromBuf
    self._tar.addfile(tarinfo=ti, fileobj=dataStream)
  File "/usr/lib64/python2.6/tarfile.py", line 2010, in addfile
    copyfileobj(fileobj, self.fileobj, tarinfo.size)
  File "/usr/lib64/python2.6/tarfile.py", line 282, in copyfileobj
    dst.write(buf)
  File "/usr/lib64/python2.6/gzip.py", line 197, in write
    self.fileobj.write( self.compress.compress(data) )
IOError: [Errno 28] No space left on device

19:41:20: Gathering output from /usr/bin/free    Exception in thread Thread-18:
Traceback (most recent call last):
  File "/usr/lib64/python2.6/threading.py", line 532, in __bootstrap_inner
    self.run()
  File "/usr/lib/vmware/site-packages/vmsupport/vmtar.py", line 187, in run
    self.runInt()
  File "/usr/lib/vmware/site-packages/vmsupport/vmtar.py", line 157, in runInt
    self._tarObj.addFromBuf(self._tarPath + suffix, dataChunk)
  File "/usr/lib/vmware/site-packages/vmsupport/vmtar.py", line 373, in addFromBuf
    self._archiver.addFromBuf(self.generateUniqDestinationPath(name), data, mode)
  File "/usr/lib/vmware/site-packages/vmsupport/archiver.py", line 103, in addFromBuf
    self._tar.addfile(tarinfo=ti, fileobj=dataStream)
  File "/usr/lib64/python2.6/tarfile.py", line 2013, in addfile
    self.fileobj.write(NUL * (BLOCKSIZE - remainder))
  File "/usr/lib64/python2.6/gzip.py", line 197, in write
    self.fileobj.write( self.compress.compress(data) )
IOError: [Errno 28] No space left on device

19:41:46: Gathering output from /usr/bin/find /sys/fs/cgroup/ -type f -print -exec cat {} ;    Exception in thread Thread-33:
Traceback (most recent call last):
  File "/usr/lib64/python2.6/threading.py", line 532, in __bootstrap_inner
    self.run()
  File "/usr/lib/vmware/site-packages/vmsupport/vmtar.py", line 187, in run
    self.runInt()
  File "/usr/lib/vmware/site-packages/vmsupport/vmtar.py", line 157, in runInt
    self._tarObj.addFromBuf(self._tarPath + suffix, dataChunk)
  File "/usr/lib/vmware/site-packages/vmsupport/vmtar.py", line 373, in addFromBuf
    self._archiver.addFromBuf(self.generateUniqDestinationPath(name), data, mode)
  File "/usr/lib/vmware/site-packages/vmsupport/archiver.py", line 103, in addFromBuf
    self._tar.addfile(tarinfo=ti, fileobj=dataStream)
  File "/usr/lib64/python2.6/tarfile.py", line 2010, in addfile
    copyfileobj(fileobj, self.fileobj, tarinfo.size)
  File "/usr/lib64/python2.6/tarfile.py", line 282, in copyfileobj
    dst.write(buf)
  File "/usr/lib64/python2.6/gzip.py", line 197, in write
    self.fileobj.write( self.compress.compress(data) )
IOError: [Errno 28] No space left on device

19:41:57: Adding /usr/lib/vmware-vmca/share/config/vmca.reg  WARNING:root:Failed to run cmd: reg query HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\services\VMwareAfdService /s: [Errno 2] No such file or directory
19:44:08: Adding /etc/vmware/vmware-vpx-workflow/def/PowerOnVM_Task.xml  ERROR:root:vc-support encounted an exception                             
Traceback (most recent call last):
  File "/bin/vc-support", line 341, in <module>
    main()
  File "/bin/vc-support", line 319, in main
    tarObj.close()
  File "/usr/lib/vmware/site-packages/vmsupport/vmtar.py", line 464, in close
    self._archiver.close()
  File "/usr/lib/vmware/site-packages/vmsupport/archiver.py", line 158, in close
    self._tar.close()
  File "/usr/lib64/python2.6/tarfile.py", line 1778, in close
    self.fileobj.close()
  File "/usr/lib64/python2.6/gzip.py", line 312, in close
    self.fileobj.write(self.compress.flush())
IOError: [Errno 28] No space left on device
Exception zlib.error: 'Error -2 while flushing: inconsistent stream state' in <bound method GzipFile.__del__ of <gzip open file u'/storage/log/vc-bart-2017-11-01--19.38.tgz', mode 'wb' at 0x99f6f0 0xa125a8>> ignored
```

**Comment by mhagen-vmware** [comment]
https://onevmw-my.sharepoint.com/personal/mhagen_vmware_com/_layouts/15/guestaccess.aspx?docid=0f19e9e7378704e86a1e2a8725b562d19&authkey=AZ0Ww30W1YQ50vtVCOGOhiU&expiration=2017-11-11T21%3a19%3a55.000Z

**Comment by mlh78750** [comment]
Is this related to bug 2005701 that we are also tracking? Can we get someone to look at this issue and triage.

**Comment by mhagen-vmware** [comment]
https://onevmw-my.sharepoint.com/personal/mhagen_vmware_com/_layouts/15/guestaccess.aspx?guestaccesstoken=VID9iCz9Ly3p3ggZ8sI7oynZAgTQJs2JLHgvSkD5uY4%3d&docid=2_1f19e9e7378704e86a1e2a8725b562d19&rev=1

**Comment by lcastellano** [comment]
I have checked the logs for this failure. There does not seem anything that would point out to the cause of the failure. Memory is a little low but not critically so. All the other OS parameters seem ok. The only anomaly is the fact that some of the service that would normally be running are "not running". This could potentially suggest that as the system was running out of memory, Linux started killing processes. This is pre photon system that uses /etc/init.d and it appears that some of these service would not restart automatically after a failure. However, the kernel log file dmesg saved at the time of the failure detection and with timestamp adjusted does not contain any messages regarding memory stress in the few hours before it was collected on the day of the failure. If the memory stress situation had occurred early in time, it is possible for the kernel log to have wrapped around. In that case the memory stress entries would be missing.  I also tried to manually shut down (on bart) the services that appear not to run at the time of the log collection. Still the system appear to be responsive for both govc and UI access. Finally, I ran the longevity test overnight and it succeeded. I will run it again today while I am looking at the rest of the logs.

**Comment by lcastellano** [comment]
After manually stopping the Component Manager and the Workflow Manager I am getting this error message when trying to log in, the longevity test is running in the background:

![screen shot 2017-12-07 at 3 20 25 pm](https://user-images.githubusercontent.com/7454168/33743718-d724dd86-db62-11e7-9719-9ebad96dce8a.png)

The vsphere-client current log shows this failure when trying to log in:

[2017-12-07T23:18:02.615Z] [ERROR] http-bio-9090-exec-42        70098446 101375 200844 com.vmware.vise.security.DefaultAuthenticationProvider            Unknown error during authentication com.vmware.vise.vim.cm.ComponentManagerConnectionException: Cannot connect to the VMware Component Manager https://bart.eng.vmware.com/cm/sdk?hostid=fe625292-ec0c-403c-b7ff-9fd6b0827084
	at com.vmware.vise.vim.cm.impl.CmUtil.handleException(CmUtil.java:38)
	at com.vmware.vise.vim.cm.impl.ComponentManagerServiceImpl.search(ComponentManagerServiceImpl.java:188)
	at com.vmware.vise.vim.cm.impl.ComponentManagerServiceImpl.search(ComponentManagerServiceImpl.java:148)
	at sun.reflect.GeneratedMethodAccessor286.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:319)
	at org.eclipse.gemini.blueprint.service.importer.support.internal.aop.ServiceInvoker.doInvoke(ServiceInvoker.java:56)
	at org.eclipse.gemini.blueprint.service.importer.support.internal.aop.ServiceInvoker.invoke(ServiceInvoker.java:60)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
	at org.springframework.aop.support.DelegatingIntroductionInterceptor.doProceed(DelegatingIntroductionInterceptor.java:131)
	at org.springframework.aop.support.DelegatingIntroductionInterceptor.invoke(DelegatingIntroductionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
	at org.eclipse.gemini.blueprint.service.importer.support.LocalBundleContextAdvice.invoke(LocalBundleContextAdvice.java:57)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
	at org.springframework.aop.support.DelegatingIntroductionInterceptor.doProceed(DelegatingIntroductionInterceptor.java:131)
	at org.springframework.aop.support.DelegatingIntroductionInterceptor.invoke(DelegatingIntroductionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
	at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:202)
	at com.sun.proxy.$Proxy188.search(Unknown Source)
	at com.vmware.vsphere.client.security.impl.ComponentManagerVcSource.fetchServicesFromCM(ComponentManagerVcSource.java:257)
	at com.vmware.vsphere.client.security.impl.ComponentManagerVcSource.getServiceInfos(ComponentManagerVcSource.java:227)
	at com.vmware.vsphere.client.security.impl.ComponentManagerVcSource.getVcs(ComponentManagerVcSource.java:58)
	at com.vmware.vsphere.client.security.VimAuthenticationHandler.getVcsWithEndpoints(VimAuthenticationHandler.java:234)
	at com.vmware.vsphere.client.security.VimAuthenticationHandler.authenticate(VimAuthenticationHandler.java:171)
	at sun.reflect.GeneratedMethodAccessor1051.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:319)
	at org.eclipse.gemini.blueprint.service.importer.support.internal.aop.ServiceInvoker.doInvoke(ServiceInvoker.java:56)
	at org.eclipse.gemini.blueprint.service.importer.support.internal.aop.ServiceInvoker.invoke(ServiceInvoker.java:60)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
	at org.springframework.aop.support.DelegatingIntroductionInterceptor.doProceed(DelegatingIntroductionInterceptor.java:131)
	at org.springframework.aop.support.DelegatingIntroductionInterceptor.invoke(DelegatingIntroductionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
	at org.eclipse.gemini.blueprint.service.importer.support.LocalBundleContextAdvice.invoke(LocalBundleContextAdvice.java:57)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
	at org.springframework.aop.support.DelegatingIntroductionInterceptor.doProceed(DelegatingIntroductionInterceptor.java:131)
	at org.springframework.aop.support.DelegatingIntroductionInterceptor.invoke(DelegatingIntroductionInterceptor.java:119)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:172)
	at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:202)
	at com.sun.proxy.$Proxy254.authenticate(Unknown Source)
	at com.vmware.vise.security.DefaultAuthenticationProvider.authenticate(DefaultAuthenticationProvider.java:234)
	at org.springframework.security.authentication.ProviderManager.authenticate(ProviderManager.java:156)
	at com.vmware.vise.security.websso.WebssoAuthenticationProcessingFilter.attemptAuthentication(WebssoAuthenticationProcessingFilter.java:47)
	at org.springframework.security.web.authentication.AbstractAuthenticationProcessingFilter.doFilter(AbstractAuthenticationProcessingFilter.java:195)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:105)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
	at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:65)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:70)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
	at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:192)
	at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:166)
	at org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:346)
	at org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:259)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:748)
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:486)
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:411)
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:338)
	at com.vmware.vsphere.client.security.websso.LogonProcessorImpl.authenticationSuccess(LogonProcessorImpl.java:176)
	at com.vmware.identity.websso.client.endpoint.SsoResponseListener.consumeResponse(SsoResponseListener.java:93)
	at com.vmware.vsphere.client.security.websso.WebssoAssertionHandler.handleRequest(WebssoAssertionHandler.java:38)
	at org.springframework.web.context.support.HttpRequestHandlerServlet.service(HttpRequestHandlerServlet.java:67)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:848)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:83)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:76)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:330)
	at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.invoke(FilterSecurityInterceptor.java:118)
	at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.doFilter(FilterSecurityInterceptor.java:84)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
	at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:113)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
	at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:103)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
	at org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFilter.java:113)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
	at org.springframework.security.web.authentication.rememberme.RememberMeAuthenticationFilter.doFilter(RememberMeAuthenticationFilter.java:139)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
	at org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter.doFilter(SecurityContextHolderAwareRequestFilter.java:54)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
	at org.springframework.security.web.savedrequest.RequestCacheAwareFilter.doFilter(RequestCacheAwareFilter.java:45)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
	at org.springframework.security.web.authentication.AbstractAuthenticationProcessingFilter.doFilter(AbstractAuthenticationProcessingFilter.java:183)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:105)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
	at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:87)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
	at com.vmware.vise.security.websso.SecurityRequestWrapperFilter.doFilterInternal(SecurityRequestWrapperFilter.java:47)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:76)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
	at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:192)
	at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:160)
	at org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:346)
	at org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:259)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at com.vmware.vise.security.SessionManagementFilter.doFilterInternal(SessionManagementFilter.java:82)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:76)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at com.vmware.vsphere.client.logging.MDCLogFilter.doFilterInternal(MDCLogFilter.java:41)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:76)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at com.vmware.vise.extensionfw.DeploymentFilter.doFilter(DeploymentFilter.java:35)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at com.vmware.vise.util.jsp.JspFilter.doFilterInternal(JspFilter.java:64)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:76)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:505)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
	at org.eclipse.virgo.web.tomcat.support.ApplicationNameTrackingValve.invoke(ApplicationNameTrackingValve.java:33)
	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:950)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:423)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1079)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:620)
	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:318)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Unknown Source)

It looks like the Component Manager must be running to do a successful web-client authentication. However govc appears to be working fine.

**Comment by mhagen-vmware** [comment]
I would think the govc issue was more related to this one than the component or workload managers:
vAPI Endpoint is not running.

**Comment by lcastellano** [comment]
The services that were not reported running are probably a red herring. There seems to be a misalignment between the services reporting their state and the actual running processes. It looks like the Component Manager and Workflow processes were actually running at the time we collected the logs. In addition it appears like their "wrapper" process was also running and it would have restarted them in case of failure. I have no theory on how the "service" representation and the actual process state got out of sync. I have rebooted the VC VM and made sure all the services were ok. Then I started the longevity test again on Friday morning. I checked on Saturday  but the Jenkins VM locked up so the result is inconclusive. At first the VC appear to be unresponsive but then I was able to log into it. I collected the logs and I am in the process of analyzing them. There are several instances of VC missing heartbeats to the hosts. The longest appear to be 36 missing heartbeats from the host: milhouse.eng.vmware.com.
